src/elements/expandable-list/expandable-list.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect, useRef } from 'react'
+
+import styles from './expandable-list.css'
+
+import { Button } from 'elements'
+import { classNames } from 'utils'
+
+const ExpandableListItem = ({ children, ...htmlProps }) => (
+  <li {...htmlProps}>{children}</li>
+)
+
+const ExpandableList = ({ children, className, ...htmlProps }) => {
+  const showMoreRef = useRef(null)
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  useEffect(() => {
+    if (isExpanded) {
+      try {
+        showMoreRef.current.parentElement.nextElementSibling.firstChild.focus()
+      } catch (e) {
+        // ignore error when element is not focusable,
+        // i.e. element is for example just text node
+      }
+    }
+  }, [isExpanded])
+
+  const childrenArray = React.Children.toArray(children)
+
+  if (childrenArray.length <= 4) {
+    return (
+      <ol
+        className={classNames.use(styles.list).join(className)}
+        {...htmlProps}
+      >
+        {childrenArray}
+      </ol>
+    )
+  }
+
+  return (
+    <ol className={classNames.use(styles.list).join(className)} {...htmlProps}>
+      {React.cloneElement(childrenArray[0], {
+        'aria-setsize': childrenArray.length,
+      })}
+      {React.cloneElement(childrenArray[1], {
+        'aria-setsize': childrenArray.length,
+      })}
+      <li
+        aria-setsize={childrenArray.length}
+        className={classNames.use(styles.showMore, isExpanded && styles.hide)}
+      >
+        <Button
+          ref={showMoreRef}
+          onFocus={() => {
+            setIsExpanded(true)
+          }}
```

Comment:
For some reason, this doesn't work in Safari despite it works in Chrome. Perhaps, the button doesn't get focus on click and maybe `onClick` is needed too.

I would add `useCallback` here if you go to add `onClick` too.